frontend/packages/data-portal/app/components/Filters/AnnotationObjectShapeTypeFilter.tsx

```diff
@@ -1,16 +1,25 @@
 import { useMemo } from 'react'
 
+import { shapeTypeToI18nKey } from 'app/constants/objectShapeTypes'
 import { QueryParams } from 'app/constants/query'
 import { useFilter } from 'app/hooks/useFilter'
 import { useI18n } from 'app/hooks/useI18n'
 import { BaseFilterOption } from 'app/types/filter'
+import { I18nKeys } from 'app/types/i18n'
+import { ObjectShapeType } from 'app/types/shapeTypes'
 
 import { SelectFilter } from './SelectFilter'
 
 function getObjectShapeTypeOptions(
   objectShapeTypes: string[],
+  t: ReturnType<typeof useI18n>['t'],
 ): BaseFilterOption[] {
-  return objectShapeTypes.map((value) => ({ value }))
+  return objectShapeTypes.map((value) => ({
+    label: t(
+      shapeTypeToI18nKey.get(value as ObjectShapeType) ?? ('' as I18nKeys),
```

Comment:
what does `t('')` actually return? If it errors might be better to put it outside of the `t()`

Reply:
yeah maybe ill make a util function for it